prow/cmd/deck/main.go

```diff
@@ -97,6 +106,16 @@ func prodOnlyMain(logger *logrus.Entry, mux *http.ServeMux) {
 	}
 	kc.SetHiddenReposProvider(func() []string { return configAgent.Config().Deck.HiddenRepos }, *hiddenOnly)
 
+	githubOAuthConfigRaw, err := loadToken(*githubOAuthConfigFile)
```

Comment:
You need to read these in the if clause you are adding below.

Reply:
that's right. My bad